app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java

```diff
@@ -0,0 +1,218 @@
+package swati4star.createpdf.fragment;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.content.pm.PackageManager;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Environment;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.support.v7.preference.PreferenceManager;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.aspose.cells.FileFormatType;
+import com.aspose.cells.PdfCompliance;
+import com.aspose.cells.PdfSaveOptions;
+import com.aspose.cells.Workbook;
+import com.dd.morphingbutton.MorphingButton;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import swati4star.createpdf.R;
+import swati4star.createpdf.util.Constants;
+import swati4star.createpdf.util.FileUtils;
+import swati4star.createpdf.util.MorphButtonUtility;
+import swati4star.createpdf.util.PermissionsUtils;
+import swati4star.createpdf.util.StringUtils;
+
+import static android.app.Activity.RESULT_OK;
+import static swati4star.createpdf.util.Constants.STORAGE_LOCATION;
+import static swati4star.createpdf.util.DialogUtils.createOverwriteDialog;
+import static swati4star.createpdf.util.StringUtils.getDefaultStorageLocation;
+import static swati4star.createpdf.util.StringUtils.getSnackbarwithAction;
+import static swati4star.createpdf.util.StringUtils.showSnackbar;
+
+public class ExceltoPdfFragment extends Fragment {
+    private Activity mActivity;
+    private FileUtils mFileUtils;
+    private Uri mExcelFileUri;
+    private String mFileExtension;
+
+    @BindView(R.id.tv_excel_file_name_bottom)
+    TextView mTextView;
+    @BindView(R.id.create_excel_to_pdf)
+    MorphingButton mCreateExcelPdf;
+
+    private SharedPreferences mSharedPreferences;
+    private MorphButtonUtility mMorphButtonUtility;
+    private static final int PERMISSION_REQUEST_WRITE_EXTERNAL_STORAGE_RESULT = 1;
+    private boolean mPermissionGranted = false;
+    private int mButtonClicked = 0;
+    private final int mFileSelectCode = 0;
+
+    public ExceltoPdfFragment() {
+    }
```

Comment:
What's the point of this blank constructor? Java automatically creates a constructor without any arguments in case you don't have one. So you can safely remove this.

Reply:
Yes will remove.

Reply:
But before that please help me with the functioning of the app. I shall make a final push removing these formatting changes.

Reply:
Do submit a push. It becomes easier to review PR if it contains only the required changes